src/services/PaymentService.ts

```diff
@@ -341,8 +341,21 @@ export class PaymentService {
    *  customer.invoice_settings.default_payment_method that precedence over
    *  customer.default_source
    */
-  async getDefaultPaymentMethod(customerId: string): Promise<PaymentMethod | CustomerSource | null> {
-    const subscriptions = await this.getActiveSubscriptions(customerId);
+  async getDefaultPaymentMethod(
+    customerId: CustomerId,
+    subscriptionType: 'individual' | 'business' = 'individual',
+  ): Promise<PaymentMethod | CustomerSource | null> {
+    if (!customerId)
```

Comment:
Remove this, as the customerId type indicates that it is not null. Any additional validation should be done in the controller.